build.gradle.kts

```diff
@@ -63,6 +63,7 @@ tasks.register("ciPublishRelease") {
 tasks.register("ciTestsGradle") {
   description = "Execute the Gradle tests (slow)"
   dependsOn(":apollo-gradle-plugin:allTests")
+  dependsOn(":apollo-gradle-plugin-external:validatePlugins")
```

Comment:
Might be worth doing this instead? 

```
dependsOn(":apollo-gradle-plugin:build")
```

I would expect `validatePlugins` to be a dependency of `build` so adding the dependency on `build` is a shortcut to "build and test everything" ? 

IIRC, the only reason this wasn't made for `ciTestsNoGradle` is to **not** execute the iOS sim tests

Reply:
We might want to partition the tests. Something like:

```
tasks.register("ciTestsGradle") {
  description = "Execute the Gradle tests (slow)"
  dependsOn(":apollo-gradle-plugin:build")
  dependsOn(":apollo-gradle-plugin-external:build")
}

tasks.register("ciTestsNoGradle") {
  subprojects {
    if (name != "apollo-gradle-plugin" && name != "apollo-gradle-plugin-external") {
      dependsOn(tasks.matching { it.name == "test" })
    }
    dependsOn(tasks.matching { it.name == "jvmTest" })
    dependsOn(tasks.matching { it.name == "jsIrTest" })
    dependsOn(tasks.matching { it.name == "macosX64Test" })
    dependsOn(tasks.matching { it.name == "apiCheck" })
  }
}
```

But this is more involved so 👍 to merge as is.